public/lib/css/index.js

```diff
@@ -0,0 +1,8 @@
+import React, {StyleSheet, Dimensions, PixelRatio} from "react-native";
+const {width, height, scale} = Dimensions.get("window"),
```

Comment:
Is const an ES6 thing? Seems like it might not be supported for all mobile browsers